test/e2e/cli-install-uninstall/cli-install-operator/operator.yaml

```diff
@@ -0,0 +1,21 @@
+name: "cli-install-operator"
```

Comment:
should we just use first-operator here? There does not seem to be anything special about this operator...

Reply:
Agreed, let's stick with something that's already there instead of copying something from the integration tests.